### DISCUSSION TITLE
Can't get verified because the domain for the school is incorrect.

### DISCUSSION THREAD

**Discussion by evandrosouza89** [community/community#49457] [GitHub Education]
When applying, I receive the following error:

> We require applicants to use a school-issued email address to apply.

And these are the domain options available for my school:

> We require applicants of Federal University of Technology Paraná to use one of these school-issued email addresses to apply:
> - professores.utfpr.edu.br
> - utfpr.edu.br
> Please select a different email address.

However, my school email domain for students is located at @alunos.utfpr.edu.br

Is there a way to get this updated?

**Comment by samus-aran** [comment]
Hi @evandrosouza89  

We have updated these school's details. You should be able to select this school as well your campus - we recommend you specifically select the campus. Please try and re-apply 🙂 

Let us know how you get on 👍
